view/frontend/web/js/model/fastlane.js

```diff
@@ -183,7 +157,7 @@ define([
          * @return {Promise<void>}
          */
         buildPPCPFastlaneInstance: async function (gatewayData) {
-            await this.loadAxo();
+            this.loadAxo(gatewayData);
             await this.loadScript('bold_paypal_fastlane_hosted_fields', 'hostedFields');
```

Comment:
For PPCP, I don't know if we need to load this script and the following one if we are also loading fastlane via www.paypal.com/sdk later on. We should confirm that.

Reply:
This actually just overrides the prototype methods to re-inject the axo scripts using requirejs

Reply:
We need as it's not compatible with require js. Fortunately, it's not force loaded as ago, se we can just load it with require js without rewrites

Reply:
this.loadAxo does that, but I think this.loadScript does some sort of loading of the actual script via require no?

Reply:
With axo.js we need to rewrite loading as axo.js is loading with force parameter. Other scripts we can just load with require js and PayPal won't reload them. But, they  are still incompatible with require js and break the page if not loaded separately